int8 control_mode
float64[3] position
float64[4] orientation_wxyz
float64[6] nominal_footholds
float64[3] left_foot_pos
float64[4] left_foot_quat
float64[3] right_foot_pos
float64[4] right_foot_quat
float64[4] camera_quat
float64[6] swing_foot_init_pos
float64[2] plan_swings
float64[2] plan_stances
float64[2] swing_total_time